scripts/build_shortform.py

```
@@ -63,6 +64,71 @@
+            blocks.append("<p>" + md_inline(" ".join(para)) + "</p>")
+            para.clear()
+
+    def flush_list():
+        if items:
+            blocks.append("<ul>" + "".join(f"<li>{md_inline(i)}</li>" for i in items) + "</ul>")
+            items.clear()
+
+    for line in text.splitlines():
+        s = line.strip()
+        if not s:
+            flush_para(); flush_list()
```

Ruff (0.15.17)

[error] Multiple statements on one line (semicolon)

(E702)